docs/fsharp/get-started/install-fsharp.md

```diff
@@ -1,41 +1,42 @@
 ---
 title: Install F#
-description: Learn how to install F# based on your environment.
-ms.date: 09/05/2019
+description: Learn how to install F# in various different ways.
+ms.date: 12/20/2019
 ---
-
 # Install F\#
 
 You can install F# in multiple ways, depending on your environment.
 
 ## Install F# with Visual Studio
 
-If you're downloading [Visual Studio](https://visualstudio.microsoft.com/vs/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link) for the first time, it will first install the Visual Studio installer. Install the appropriate SKU of Visual Studio from the installer. If you already have it installed, click **Modify**.
+1. If you're downloading [Visual Studio](https://visualstudio.microsoft.com/vs/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link) for the first time, it will first install Visual Studio Installer. Install the appropriate edition of Visual Studio from the installer.
```

Comment:
should we send them here instead?

```suggestion
1. If you're downloading [Visual Studio](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019) for the first time, it will first install Visual Studio Installer. Install the appropriate edition of Visual Studio from the installer.
```